CMakeLists.txt

```diff
@@ -305,20 +316,23 @@ TARGET_LINK_LIBRARIES(facemask-plugin
 ################################################################################
 # Installation / Distribution
 ################################################################################
+if(TARGET libobs)
+	install_obs_plugin_with_data(facemask-plugin data)
+else()
 
-
-add_custom_command(TARGET facemask-plugin POST_BUILD
-	COMMAND ${CMAKE_COMMAND} -E copy_directory
-	"${PROJECT_SOURCE_DIR}/data"
-	"${DISTRIBUTE_PATH}/$<CONFIG>/data/obs-plugins/facemask-plugin"
-)
-add_custom_command(TARGET facemask-plugin POST_BUILD
-	COMMAND ${CMAKE_COMMAND} -E copy
-	"$<TARGET_FILE:facemask-plugin>"
-	"${DISTRIBUTE_PATH}/$<CONFIG>/${DISTRIBUTE_PATH_BIN}/$<TARGET_FILE_NAME:facemask-plugin>"
-)
-add_custom_command(TARGET facemask-plugin POST_BUILD
-	COMMAND ${CMAKE_COMMAND} -E copy
-	"$<TARGET_FILE_DIR:facemask-plugin>/facemask-plugin.pdb"
-	"${DISTRIBUTE_PATH}/$<CONFIG>/${DISTRIBUTE_PATH_BIN}/facemask-plugin.pdb"
-)
+	add_custom_command(TARGET facemask-plugin POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy_directory
+		"${PROJECT_SOURCE_DIR}/data"
+		"${DISTRIBUTE_PATH}/$<CONFIG>/data/obs-plugins/facemask-plugin"
+	)
+	add_custom_command(TARGET facemask-plugin POST_BUILD
```

Comment:
Please use the `install()` command for this. POST_BUILD should be avoided unless necessary.

Reply:
These lines actually  are only added tabs (that's why it shows changed)
Anyways, I can change to install command, however, every time after building *cmake install* should be run from command or VS,
Let me know if it is important to avoid POST_BUILD and I'll change it